docs/website/components/Header.vue

```diff
@@ -14,6 +14,11 @@
           <li class="link min-w-full md:min-w-0">
             <CommunityDropdown></CommunityDropdown>
           </li>
+          <li class="link min-w-full md:min-w-0">
+            <a href="/roadmap">
```

Comment:
I'd like to eventually have a "Project" dropdown, but in case we can't get to that by kubecon, this should hold us over.